.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ['3.0', '3.1', '3.2']
```

Comment:
ruby 3 is EOL as of 1 week ago ish, it seems https://endoflife.date/ruby

i think we can just do 3.1, 3.2, and 3.3